### PR TITLE
Update assign-pod-node.md

### PR DESCRIPTION
**Pull request by WanLinghao**
I am a freshman on Kubernetes development.When I use the affinity feature,it confuses me that why the pod won't stop after I changing  the node's label which makes the node no longer satisfy the  affinity condition.Although I learn  how this feature works by community, I hope the documentation can help  people especially fresh man better.^_^

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4356)

### CODE REVIEW

File: docs/concepts/configuration/assign-pod-node.md [hunk 1]
```diff
@@ -142,6 +142,8 @@ If you specify multiple `nodeSelectorTerms` associated with `nodeAffinity` types
 
 If you specify multiple `matchExpressions` associated with `nodeSelectorTerms`, then the pod can be scheduled onto a node **only if all** `matchExpressions` can be satisfied.
 
+If you remove or change the label of the node which the pod is running on,the running pod won't be stopped.It means that the affinity selection only works when the pod is scheduled.
```
Comment (bsalamat) [inline docs/concepts/configuration/assign-pod-node.md]: Please replace with:

If you remove or change the label of the node where the pod is scheduled, the pod won't be removed. In other words, the affinity selection works only at the time of scheduling the pod.

**Comment by bsalamat** [comment]
/lgtm

thanks, @WanLinghao for the improvement.

**Comment by ahmetb** [comment]
@WanLinghao can you please check the box on the right that says "Allow contributions from maintainers"? That way we can rebase and merge.

**Comment by WanLinghao** [comment]
@ahmetb sorry for that。first time to make a pull

**Comment by WanLinghao** [comment]
@ahmetb  did I miss something?no merge happens....

**Comment by ahmetb** [comment]
Sorry it was waiting someone to sync the branch to latest and just hit merge.

**Pull request merged**